managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2954,6 +2955,7 @@ private void cleanupOffloaded(long ledgerId, UUID uuid, String offloadDriverName
                                                                                        * identify offloader
                                                                                        */
             Map<String, String> offloadDriverMetadata, String cleanupReason) {
+        log.info("cleanup offloaded data uuid {} for reason {}", uuid.toString(), cleanupReason);
```

Comment:
It's better to also print the managed ledger name so that we can distinguish which topic occurs the problem. And also print the ledgerId? it's useful for debugging.

Use info level log looks good here since the cleanupOffloaded only happens while offload failed right @gaoran10

Reply:
Yes, I think the cleanup offloaded data is a low-frequency operation.